### PR TITLE
A default placeholder validator for bulk params

### PR DESCRIPTION
**Pull request by huaaaan**
We need this default `bulk_validator` to always return true so that other endpoints can still use apipie validation and bulk update endpoints can have its own custom validation strategy.

### CODE REVIEW

File: Gemfile.rails50 [hunk 1]
```diff
@@ -1,9 +1,10 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 gemspec
 
-gem 'rails', '~> 5.0.0'
 gem 'mime-types', '~> 2.99.3'
+gem 'rails', '~> 5.0.0'
 gem 'rails-controller-testing'
+gem 'sqlite3', '~> 1.3.6'
```
Comment (emmettu) [inline Gemfile.rails50:8]: are these changes intentional?

Comment (huaaaan) [reply Gemfile.rails50:8]: yup, the repo is missing this gem, o.w. the spec files cannot be run

**Review by emmettu — APPROVED** [review]
LGTM, one question: why can't we use apipie for the validation instead of writing our own?

**Comment by huaaaan** [comment]
> LGTM, one question: why can't we use apipie for the validation instead of writing our own?

Apipie validation can only return true or false and what we want is more like a filter that returns back the valid params and invalid params

File: README.rst [hunk 2]
```diff
@@ -1316,6 +1316,18 @@ description of nested values.
...
+BulkValidator
+-------------
+
+BulkValidator is only a placeholder validator which always return true for bulk params. BulkValidator is introduced because we need to bypass apipie validation for our bulk params and create its own non-apipie validation logic without touching other endpoints.
+
+.. code:: ruby
+
+   param :bulk_params, :bulk, :desc => "Bulk params" do
+     param :id, String, :desc => "Id of keyword", :required => true
+     param :keyword, String, :desc => "Keyword", :required => true
```
Comment (AzRu) [inline README.rst:1328]: what about the other ones that arent keyword?

Comment (huaaaan) [reply README.rst:1328]: This is just an example of how to use` :bulk` type validators. Any params defined as `:bulk `will just return true

Comment (AzRu) [reply README.rst:1328]: ahhh makes so much sense now.. somehow i missed that this was an example lol
